script/helperFunctions.sh

```
@@ -1441,74 +1434,72 @@ function verifyContract() {
-        fi
-      else
-        if isZkEvmNetwork "$NETWORK"; then
-          # Verify using foundry-zksync
-         FOUNDRY_PROFILE=zksync ./foundry-zksync/forge verify-contract --zksync --watch --chain "$CHAIN_ID" "$ADDRESS" "$FULL_PATH" --constructor-args $ARGS --skip-is-verified-check -e "${!API_KEY}" >/dev/null 2>&1
-        else
-          forge verify-contract --watch --chain "$CHAIN_ID" "$ADDRESS" "$FULL_PATH" --constructor-args $ARGS --skip-is-verified-check -e "${!API_KEY}" >/dev/null 2>&1
-        fi
-      fi
+  # Get API key and determine verification method
+  API_KEY=$(getEtherscanApiKeyName "$NETWORK")
+  if [ $? -eq 0 ]; then
```

Shellcheck (0.10.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)